docs/ops.html

```diff
@@ -3837,7 +3837,7 @@ <h4 class="anchor-heading"><a id="kraft_deployment" class="anchor-link"></a><a h
 
   <ul>
     <li>Kafka server's <code>process.role</code> should be set to either <code>broker</code> or <code>controller</code> but not both. Combined mode can be used in development environments, but it should be avoided in critical deployment environments.</li>
-    <li>For redundancy, a Kafka cluster should use 3 or 5 controllers, depending on factors like cost and the number of concurrent failures your system should withstand without availability impact. In the rare case of a partial network failure it is possible for the cluster metadata quorum to become unavailable. This limitation will be addressed in a future release of Kafka.</li>
+    <li>For redundancy, a Kafka cluster should use 3 or 5 controllers, depending on factors like cost and the number of concurrent failures your system should withstand without availability impact. In the rare case of a partial network failure it is possible for the cluster metadata quorum to become unavailable.</li>
```

Comment:
How about something like:

```java
    <li>For redundancy, a Kafka cluster should use 3 or more controllers, depending on factors like cost and the number of concurrent failures your system should withstand without availability impact. For the KRaft controller cluster to withstand <code>N</code> concurrent failures the controller cluster must include <code>2N + 1</code> controllers.</li>
```

Reply:
It's more clear.
I will adopt it, thank you 😄

Reply:
Preview:
![Screenshot from 2024-12-21 00-24-03](https://github.com/user-attachments/assets/6d319f3f-ab8b-4fb1-8218-49a6c000e3c4)